piserver/app.py

```diff
@@ -7,15 +7,24 @@
 
 
 from flask import Flask, request
+from flask_cors import CORS
 from piserver.api_routes import *
 from blinds.blinds_api import Blinds, SmartBlindsSystem
 
 # flask setup
 app = Flask(__name__)
 app.config['TESTING'] = True
+app.config['CORS_HEADERS'] = 'Content-Type'
+CORS(app)
 
 # INIT BLINDS SYSTEM RELATED COMPONENTS #
-smart_blinds_system =  SmartBlindsSystem( Blinds( None ), None, None )
+smart_blinds_system = None
```

Comment:
@AY-Chen would this be the best way to inject dependencies depending on whether the sensors are connected or not? Are there cases where we want to run the app in dev mode but still use the sensors?

Reply:
Yeah there are cases where we would. Id rather we keep the SmartBlindsSystem and just be able to handle cases where the sensors are mocked. Otherwise if we make the whole system None, then the api handlers will need to implement cases itself since you'll get NoneType errors.